metrics_utility/automation_controller_billing/collector.py

```diff
@@ -1,10 +1,18 @@
 import contextlib
 import json
 import logging
+import importlib.util
 
 import insights_analytics_collector as base
 from awx.main.utils import datetime_hook
-from awx.main.utils.pglock import advisory_lock
+
+if importlib.util.find_spec('ansible_base') is None:
```

Comment:
This is from Martins unfinished PR:
if importlib.util.find_spec('ansible_base') is None:
    # 2.4
    from awx.main.utils.pglock import advisory_lock
else:
    # 2.5
    from ansible_base.lib.utils.db import advisory_lock
    
    Otherwise, this PR would be blocked.